go/go.mod

```diff
@@ -1,4 +1,4 @@
-module fabric-admin-sdk
+module github.com/hyperledger-twgc/fabric-admin-sdk
```

Comment:
Does that actually work or will the module path need to be `github.com/hyperledger-twgc/fabric-admin-sdk/go`?

Reply:
well I hope it works, we as don't have go mod at https://github.com/hyperledger/fabric-admin-sdk